lib/src/app.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+
+import '../src/screens/onboarding_screen.dart';
+import '../src/screens/home_screen.dart';
+
+class App extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      routes: {
+        '/': (context) => OnboardingScreen(),
+        '/Home': (context) => HomeScreen(),
```

Comment:
/Home => /home
https://flutter.dev/docs/cookbook/navigation/named-routes

Reply:
I fixed this error.